README.md

```diff
@@ -66,6 +69,9 @@ new Siema({
 - `loop` - (boolean) loop the slides around
 - `onInit` - (function) runs immediately after first initialization
 - `onChange` - (function) runs after slide change
+- `afterPrev` - (function) runs after going to previous slide
+- `afterNext` - (function) runs after going to next slide
+- `onDestroy` - (function) runs after removing all active listeners
```

Comment:
We have to ditch afterPrev() and afterNext() from this PR. In current implementation it invokes actually straight before next() and prev(), not after it. Doing before and after is a bit more complicated as it needs to listen for transitionstart and transitionend events on instance's sliderFrame. Let's keep it for future versions of Siema.

The onDestory() callback — perfecto :)